src/Emoji/Emoji.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react'
+import styled, { css } from 'styled-components'
+
+type EmojiProps = {
+  /** className attribute to apply classes from props */
+  className?: string
+  /** Emoji symbol */
+  symbol: string
+  /** set size of the Emoji  */
+  size?: number
+  /** aria-label to identify Emoji */
+  label?: string
```

Comment:
Do you think this should be enforced rather than optional to avoid a bunch of "emoji" strings?